pkg/api/plugin.go

```diff
@@ -73,8 +73,8 @@ type PluginConfig struct {
 	SyncImage       string
 	AcceptLanguages []string
 	GenevaConfig    GenevaConfig
-
-	TestConfig TestConfig
+	IsAdmin         bool
```

Comment:
I really don't think it can be right to have `IsAdmin bool` as well as a separate ValidateAdminUpdate() entrypoint in the interface.
I'd rather remove `IsAdmin bool`, add a ValidateAdmin() in the interface instead of ValidateAdminUpdate(), have that function call `func (v *Validator) ValidateAdmin`, which can then reuse validateContainerService and have its own copy validateAdminUpdateContainerService, as you've implemented